<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgBoxClass</class>
 <widget class="QDialog" name="MsgBoxClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MsgBox</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: none; 
background-color:white;</string>
  </property>
  <widget class="QPushButton" name="msgOK">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font: 12pt &quot;幼圆&quot;;
background-color:#457fc1;
border-radius:6px;</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="msgX">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../images/BTMsgBox/wrong2.png</normaloff>../../../images/BTMsgBox/wrong2.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="msgIcon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../images/BTMsgBox/notice2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="msgContent">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>消息提示</string>
   </property>
  </widget>
  <widget class="QLabel" name="msgTitle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>标题测试</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
